### PR TITLE
Documentation Vienna

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/2091
https://github.com/FreshRSS/FreshRSS/pull/2093
https://github.com/ViennaRSS/vienna-rss/issues/1197

### CODE REVIEW

**Comment by Alkarex** [comment]
@barijaona FYI. Comments welcome.

**Comment by Alkarex** [comment]
Let's see whether there is a reaction from https://github.com/rss-sync/Open-Reader-API/issues/13
Referring to [Open Reader API](http://rss-sync.github.io/Open-Reader-API/rssconsensus/) only makes sense if it can be substantially updated in the near future.

File: README.fr.md [hunk 1]
```diff
@@ -159,18 +159,20 @@ Voir le [dépôt dédié à ces extensions](https://github.com/FreshRSS/Extensio
...
 
-Tout client supportant une API de type Google Reader ; Sélection :
+Tout client supportant une API de type Google Reader / [Open Reader](http://rss-sync.github.io/Open-Reader-API/rssconsensus/) ; Sélection :
 
 * Android
 	* [News+](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus) avec [News+ Google Reader extension](https://play.google.com/store/apps/details?id=com.noinnion.android.newsplus.extension.google_reader) (Propriétaire)
 	* [FeedMe 3.5.3+](https://play.google.com/store/apps/details?id=com.seazon.feedme) (Propriétaire)
 	* [EasyRSS](https://github.com/Alkarex/EasyRSS) (Libre, [F-Droid](https://f-droid.org/fr/packages/org.freshrss.easyrss/))
 * GNU/Linux
 	* [FeedReader 2.0+](https://jangernert.github.io/FeedReader/) (Libre)
```
Comment (Frenzie) [inline README.fr.md:182]: It doesn't work with Liferea? Admittedly I haven't tried it.

**Comment by Alkarex** [comment]
Removing references to Open Reader API for now, while waiting for a sign of life

File: README.fr.md (hunk 1 above)
Comment (Alkarex) [reply README.fr.md:182]: https://github.com/FreshRSS/FreshRSS/issues/3291

Comment (tabic) [reply README.fr.md:182]: I asked again, https://github.com/lwindolf/liferea/issues/962

**Pull request merged**